sdk_container/src/third_party/coreos-overlay/sys-kernel/bootengine/bootengine-9999.ebuild

```
@@ -10,7 +10,7 @@ CROS_WORKON_REPO="https://github.com"
 if [[ "${PV}" == 9999 ]]; then
 	KEYWORDS="~amd64 ~arm ~arm64 ~x86"
 else
-	CROS_WORKON_COMMIT="31d24b87717e6ee3b2970af5926a481d6bbff933" # flatcar-master
+	CROS_WORKON_COMMIT="72c39f0963f31e5dcbb262213a453916836e9099" # TODO: flatcar-master
```

```suggestion
	CROS_WORKON_COMMIT="5295e877b6ebd2fb343e464ffda896534c22d4c3" # flatcar-master
```
Ok, I've merged the PR

---

(For the backport we'll use `flatcar-3760-backport` but that happens after merging this scripts PR)